### DISCUSSION TITLE
de/2-05-kata-4/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#63] [General]
# de/2-05-kata-4/

https://ai.lernos.org/de/2-05-kata-4/

**Comment by rednaxelaesor** [comment]
Personas - Es gibt sooo viele ausgezeichnete Autoren. Joane K. Rowling würde ich aufgrund ihrer offenen transfeindlichen Haltung nicht unbedingt aufführen. Natürlich trenne ich zwischen Werk und Autor, dennoch glaube ich das diese Empfehlung nicht im vollen Einklang des Geistes und den Grundwerten  von lernOS steht. IMHO

**Reply by fdenzer** [reply]
"Schreibe einen Tweet im Ton von J.K. Rowling" ist *keine* gute Idee, dem stimme ich zu.

**Comment by Flesch-04-18** [comment]
Die Persona kann auch als Rollenbeschreibung und nicht als Stilmittel genutzt werden.

Kannst du mir Methode XY aus der sicht eines Scrummasters / Agile Coach erklären.


Was besonders gut für Promptgenerierung genutzt werden kann, wenn einem das experten Wissen in Bereichen fehlt.

Erstelle mir einen Prompt der aus sicht eines Produktfotografen ...

**Comment by stefandiepolder** [comment]
Ich habe eine Frage. Würdet ihr unter Personas auch die Custom Instructions (im Beispiel ChatGPT) verstehen? Ich finde dieses Video von den Digitalen Profis sehr hilfreich, um der KI zu sagen, wer sie sein soll (Persona) und was sie genau tun soll und um ein Verständnis zu bekommen, warum diese Kontexte wichtig sin - https://youtu.be/EM9sSQxnyfk - 13 Minuten
